### PR TITLE
[Bot-Solutions] [TypeScript] Fix MultiProviderAuthDialog localization

### PR DESCRIPTION
**Pull request by VictorGrycuk**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Review by Batta32 — APPROVED** [review]
All the changes LGTM!!

File: sdk/typescript/libraries/bot-solutions/src/authentication/multiProviderAuthDialog.ts [hunk 1]
```diff
@@ -65,29 +66,35 @@ export class MultiProviderAuthDialog extends ComponentDialog {
...
 
-            this.addDialog(new WaterfallDialog(DialogIds.firstStepPrompt, authSteps));
+            this.addDialog(new WaterfallDialog(DialogIds.authPrompt, authSteps));
             this.addDialog(new ChoicePrompt(DialogIds.providerPrompt));
         } else {
             throw new Error('There is no authenticationConnections value');
         }
     }
 
+    private getLocalizedDialog(locale: string, connectionName: string, settings: OAuthPromptSettings) {
```
Comment (fran893) [inline sdk/typescript/libraries/bot-solutions/src/authentication/multiProviderAuthDialog.ts:81]: I don't remember about TS, but here, don't we need to add the return type in the method signature?

**Review by fran893 — CHANGES_REQUESTED** [review]

File: sdk/typescript/libraries/bot-solutions/src/authentication/multiProviderAuthDialog.ts (hunk 1 above)
Comment (Batta32) [reply sdk/typescript/libraries/bot-solutions/src/authentication/multiProviderAuthDialog.ts:81]: +1, @VictorGrycuk have you tested the linter/build/tests?

File: sdk/typescript/libraries/bot-solutions/src/authentication/multiProviderAuthDialog.ts [hunk 2]
```diff
@@ -65,29 +66,35 @@ export class MultiProviderAuthDialog extends ComponentDialog {
...
-                        this.authPromptValidator.bind(this)
-                    ));
-                }
+                MultiProviderAuthDialog.acceptedLocales.forEach(locale => {
+                    this.addDialog(this.getLocalizedDialog(locale, connection.name, promptSettings[i]));
+                });
             };
 
-            this.addDialog(new WaterfallDialog(DialogIds.firstStepPrompt, authSteps));
+            this.addDialog(new WaterfallDialog(DialogIds.authPrompt, authSteps));
```
Comment (matiasroldan6) [inline sdk/typescript/libraries/bot-solutions/src/authentication/multiProviderAuthDialog.ts:74]: I think we already have this change somewhere else, in 1.0/next? But it's ok

Comment (VictorGrycuk) [reply sdk/typescript/libraries/bot-solutions/src/authentication/multiProviderAuthDialog.ts:74]: Several changes are already in next, but we need this in master as it is a bugfix

File: sdk/typescript/libraries/bot-solutions/src/authentication/multiProviderAuthDialog.ts (hunk 1 above)
Comment (VictorGrycuk) [reply sdk/typescript/libraries/bot-solutions/src/authentication/multiProviderAuthDialog.ts]: Feedback applied!

Comment (Batta32) [reply sdk/typescript/libraries/bot-solutions/src/authentication/multiProviderAuthDialog.ts]: LGTM

**Pull request merged**